vendor/github.com/go-rod/rod/lib/js/helper.js

```
@@ -0,0 +1,572 @@
+//     go test -run ^TestClick$ -- -rod=show,devtools
+
+const functions = {
+  element(selector) {
+    const s = functions.selectable(this)
+    return s.querySelector(selector)
+  },
+
+  triggerFavicon() {
+    return new Promise((resolve, reject) => {
+      const faviconElement = document.querySelector('link[rel~=icon]')
+      const href = (faviconElement && faviconElement.href) || '/favicon.ico'
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)